{"timestamp":"2025-10-29T13:35:28.119808","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-10-29T13:35:28.121950","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/ARPEGE.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-10-29T13:35:29.948637Z","level":"info","event":"üîÑ Descargando datos de http://openmeteo-api:8080/v1/forecast ...","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T13:35:29.948992Z","level":"info","event":"‚¨áÔ∏è Descargando temperature_2m del Bucket S3 de AWS...","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T13:35:30.001239Z","level":"error","event":"docker: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?.","chan":"stderr","logger":"task"}
{"timestamp":"2025-10-29T13:35:30.001689Z","level":"error","event":"See 'docker run --help'.","chan":"stderr","logger":"task"}
{"timestamp":"2025-10-29T13:35:30.007650","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"CalledProcessError","exc_value":"Command '['docker', 'run', '--rm', '-v', 'open-meteo-data:/app/data', 'ghcr.io/open-meteo/open-meteo', 'sync', 'meteofrance_arpege_europe', 'temperature_2m']' returned non-zero exit status 125.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1215,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/ARPEGE.py","lineno":165,"name":"ejecutar_descarga_arpege"},{"filename":"/opt/airflow/dags/ARPEGE.py","lineno":78,"name":"fetch_forecast"},{"filename":"/usr/local/lib/python3.12/subprocess.py","lineno":571,"name":"run"}],"is_group":false,"exceptions":[]}]}
